### ISSUE TITLE
NES PPU Viewer's "zoom box" shouldn't be anti-aliased

### ISSUE THREAD

**Issue by YoshiRulz** [TASEmulators/BizHawk#3764]
This guy:
https://github.com/TASEmulators/BizHawk/blob/ebe3cad66c6989a7e29b9ee77ca7e426707319a2/src/BizHawk.Client.EmuHawk/tools/NES/NESPPU.Designer.cs#L726
`PictureBox` doesn't accept an `InterpolationMode` though. [This SO answer](https://stackoverflow.com/a/13484101) suggests subclassing it and overriding `OnPaint` to mutate the `Graphics` before calling the base method. Don't forget to set `PixelOffsetMode = Half` too if you go ahead with this.

**Comment by Morilli** [comment]
Is there a bug currently? From what I see, the graphics in that box are not antialised, so what is the suggestion here?

**Comment by YoshiRulz** [comment]
Still present at f309dbc31. This may be Mono-specific, I recall @TalicZealot had a similar bug in their ext. tool on Linux only.
![screenshot](https://github.com/TASEmulators/BizHawk/assets/13409956/c1d59b88-097d-4f1c-a12a-3b20260e5517)

**Comment by Morilli** [comment]
> This may be Mono-specific

Yep, looks that way. I found some old comments on `InterpolationMode.NearestNeighbor` being ignored, and it has something to do with Mono's libgdiplus using Cairo and that handles stuff differently... or something. In any case, this is most likely not related to anything BizHawk at all.
